### DISCUSSION TITLE
Reading (Partial Strings) into char[]

### DISCUSSION THREAD

**Discussion by raddy** [stephenberry/glaze#264] [Q&A] [answered]
If we've got a struct:
```
struct A
{
    int  x;
    char buf[12];
};
```

And we've got some JSON: `std::string buffer = R"({"x":"999", "some_string": "SHORT_STRING_NAME"})";`

Is there a way to read directly into the struct's `buf`? (I guess writing that which fits + null term)

**Comment by stephenberry** [comment]
You should be able to read into a std::string_view, so I think the following would work: `std::string_view buf = "xxxxxxxxxxxx";`
But, I think we should allow support for this use case. I'll add an issue.

**Comment by stephenberry** [comment, marked as answer]
Feature was added.
